docs/source/exercises.rst

```diff
@@ -75,47 +75,41 @@ However, things change dramatically when we allow for quantum strategies leading
 
 Can you design a program that simulates this game?
 
-Examples of Quantum Programs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To create intuition for a new class of algorithms, that will run on the Quantum Virtual Machine (QVM), it is useful (and
-fun) to play with the abstraction that the software provides.
+The Meyer-Penny Game
+~~~~~~~~~~~~~~~~~~~~
 
-A broad class of programs that can easily be implemented on a QVM are generalizations of
-`Game Theory to incorporate Quantum Strategies <https://arxiv.org/abs/quant-ph/0611234>`_.
+To create intuition for quantum algorithms, it is useful (and fun) to play with the abstraction that
+the software provides.
 
-Meyer-Penny Game
-----------------
-
-A conceptually simple example that falls into this class is the
-`Meyer-Penny Game <https://link.aps.org/doi/10.1103/PhysRevLett.82.1052>`_. The game goes as follows:
+A simple example from game theory is the Meyer-Penny Game [1]_. The interested reader may want to read more about
+quantum game theory in the article *Toward a general theory of quantum games* [2]_. The Meyer-Penny Game goes as follows:
 
 The Starship Enterprise, during one of its deep-space missions, is facing an immediate calamity, when a powerful alien
 suddenly appears on the bridge. The alien, named Q, offers to help Picard, the captain of the Enterprise, under the
```

Comment:
what bridge?!

Reply:
[*the* bridge](https://en.wikipedia.org/wiki/Bridge_(nautical))